test/test-ParameterParser.cc

```diff
@@ -141,6 +141,17 @@ TEST_CASE("Recognize parameter with brackets in old syntax example value", "[par
     REQUIRE(sectionType == ParameterSectionType);
 }
 
+TEST_CASE("Recognize escaped parameter with brackets in old syntax example value", "[parameter]")
+{
+    mdp::MarkdownParser markdownParser;
+    mdp::MarkdownNode markdownAST;
+    markdownParser.parse("+ `id` (optional, oData, `example`)", markdownAST);
+
+    REQUIRE(!markdownAST.children().empty());
+    SectionType sectionType = SectionProcessor<Parameter>::sectionType(markdownAST.children().begin());
+    REQUIRE(sectionType == ParameterSectionType);
```

Comment:
It does run that code for old parameters.

```apib
# Resource [/test{?x_y}]

+ Parameters
  + `x_y` (required, string, `value`)

## Action [GET]

+ Response 204
```

```json
{
  "href": "\/test{?x_y}",
  "hrefVariables": {
    "element": "hrefVariables",
    "content": [
      {
        "element": "member",
        "attributes": {
          "typeAttributes": [
            "required"
          ]
        },
        "content": {
          "key": {
            "element": "string",
            "content": "x_y"
          },
          "value": {
            "element": "string",
            "content": "value"
          }
        }
      }
    ]
  }
}
```

![screen shot 2016-10-13 at 11 59 06](https://cloud.githubusercontent.com/assets/44164/19346654/790e4578-913c-11e6-8570-1b6c480a8eed.png)